packages/flutter/lib/src/material/autocomplete.dart

```diff
@@ -67,22 +67,23 @@ class Autocomplete<T extends Object> extends StatelessWidget {
   /// default.
   final AutocompleteOptionsViewBuilder<T>? optionsViewBuilder;
 
-  /// The maximum height used for the default Material options list widget.
+  /// The constraints of the suggestions Overlay.
   ///
-  /// When [optionsViewBuilder] is `null`, this property sets the maximum height
-  /// that the options widget can occupy.
-  ///
-  /// The default value is set to 200.
-  final double optionsMaxHeight;
+  /// The default height constraint is set to 200
```

Comment:
Period at the end of the sentence here, and either make the following sentence part of this paragraph or add an empty `///` line between them.